Sources/Bot/Services/GitHubService.swift

```diff
@@ -43,10 +50,37 @@ public final class GitHubService {
             return SignalProducer(error: .unknown)
         }
     }
+
+    private static func signatureVerifier(
+    ) -> (Token) -> (RequestProtocol) -> Result<RequestProtocol, EventHandlingError> {
+
+        return { token in
+            return { request in
+                guard
+                    let signature = request.header(.signature),
+                    let digest = signature.range(of: "sha1=")
+                        .map({ signature[$0.upperBound..<signature.endIndex] })
+                        .map(String.init)
+                        .map(Array.init(hex:))
```

Comment:
🤔❔ Where does `Array(hex:)` comes from, is that provided by `CryptoSwift`?

Reply:
Yeah, CryptoSwift's sugar.